packages/fxa-js-client/tests/mocks/request.js

```diff
@@ -26,6 +26,10 @@ define(['client/lib/errors', 'tests/lib/push-constants'], function(
       headers: {},
       body: '{}',
     },
+    deleteSecurityEvents: {
+      status: 200,
+      body: '{}',
```

Comment:
Just noticed this, but it is actually an empty array `[]` instead of empty response. Can you update the test and mock.

Reply:
No problem but it was an empty array [before](https://github.com/mozilla/fxa/pull/1848#discussion_r306831951)

Reply:
@vbudhram done.